src/index/login/LoginForm.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import axios from 'axios';
+import SweetAlert from 'react-bootstrap-sweetalert';
+import { Form, Button } from 'react-bootstrap';
+import './Login.less';
+import '../IndexForm.less';
+
+export default class LoginForm extends React.Component<any, any> {
```

Comment:
try using Hooks in functional components instead of classes, that way you won't have to bind methods in construcltor and will have access to many useful hooks like useHistory from react-router